frontend/playwright/ui/specs/tokens.spec.js

```diff
@@ -46,7 +46,7 @@ const setupTokensFile = async (page, options = {}) => {
   } = options;
 
   const workspacePage = new WorkspacePage(page);
-  if (flags.length) {
+  if (flags.length > 0) {
```

Comment:
This should not be necessary. This is the same as `flags.lenght`

Reply:
Its not necessary but is easy to understand for me.